frontend/src/app/modules/queue-dashboard/queue-dashboard.component.html

```diff
@@ -0,0 +1,34 @@
+<h1>
+  {{'frontend.de.iteratec.osm.queueDashboard.title' | translate}}
+</h1>
+<div class="card" *ngFor="let wptserver of wptServer$ | async" #thisCard isOpened="false" isLoaded="false">
```

Comment:
Since both member variables refer to the same scope (`thisCard`), you can also define `isOpened` and `isLoaded` as member variables of your component. This way you don't need to use the `#thisCard` reference and you can access the properties easily from withing the component-logic.